README.md

```diff
@@ -46,65 +32,38 @@ We first need to start a local Espresso Sequencer network.
     docker compose pull
     docker compose up -d --wait
 
-Once that is finished deploy the example rollup, it can be run natively or with docker.
-
-To run the demo inside docker run the following
-
-    docker compose pull
-    docker compose up
-
-To stop the docker demo run `docker compose down`.
+Once that is finished deploy the example rollup, it can be run natively.
 
 To run natively a rust toolchain is required. We provide a nix shell to install
 all dependencies. Nix can be installed via the
 [nix-installer](https://github.com/DeterminateSystems/nix-installer). Once
-installed run
+installed run the following from the root of the project:
 
     nix develop
     just dev-demo
```

Comment:
I would say adding the screenshots to show the user what they expect to see after a successful run, as well as indicating which log should be present before they check for the balance, would be a nice add-on and can help the user understand espresso in a more meaningful way.